src/test/java/io/patriot_framework/virtual_smart_home/route/HouseRouteTest.java

```diff
@@ -42,21 +41,4 @@ public void emptyGetRequest() throws JSONException {
                 .then().assertThat().body(Matchers.equalTo(expected.toString()));
     }
 
-    @Test
-    @DirtiesContext(methodMode = DirtiesContext.MethodMode.AFTER_METHOD)
-    public void simpleGetRequest() throws JSONException {
```

Comment:
If the test-case is failing then, rather than removing the test it is better to either fix it or disable it with `Disabled` annotation https://junit.org/junit5/docs/5.3.0/api/org/junit/jupiter/api/Disabled.html and comment the needed fix.

Reply:
This Test case is not Part of Master. So in my point of view it was a first try within the old migration. So I removed it.

Reply:
Also the Test itself was wrong, because house does not have any post function implemented. Please write a new test if you think that some test coverage is not available. It would be great if you could join #hacktoberfest ;-)